roles/common/tasks/main.yml

```diff
@@ -73,11 +73,23 @@
 - include: hwraid.yml
   when: common.hwraid.enabled
 
+
+- name: Fetch Ursula revision
+  local_action: command git describe --tags
+  run_once: true
+  register: ursula_git_describe
```

Comment:
That could be an assert earlier in the play I would think, outside of the motd setting.